### ISSUE TITLE
Reinforcement Learning Chapter 18

### ISSUE THREAD

**Issue by OrnellaFanais** [ageron/handson-ml2#217]
Hi everyone!
I would need to create a random action based on the estimated probabilities like this:
action_index = tf.squeeze(tf.multinomial(logits, num_samples=1), axis=-1) 
present in (exercise 8)
https://github.com/ageron/handson-ml/blob/master/16_reinforcement_learning.ipynb 
I'm using Tensorflow 2.0
Thanks !!!!!!!!
